test/test_mcs.py

```diff
@@ -0,0 +1,136 @@
+"""
+Regression tests for the MCS class
+
+These might start failing if RDKit changes canonical order of atoms.
+"""
+import pkg_resources
+import pytest
+from rdkit import Chem
+
+from lomap import mcs
+
+
+def _rf(fn):
+    # get path to file from inside lomap installation
+    f = pkg_resources.resource_filename('lomap', 'test/' + fn)
+    return f.replace('/lomap/test', '/test')
+
+
+@pytest.fixture
+def toluene():
```

Comment:
I'm not a massive fan of the way these are laid out - maybe they should be in conftest - but given it's how it was done before I won't block just for this.